### PR TITLE
Allow Bot to return Binary

### PR DESCRIPTION
**Pull request by codyebberson**

### CODE REVIEW

File: packages/server/src/fhir/operations/execute.test.ts [hunk 1]
```diff
@@ -394,20 +410,31 @@ exports.handler = async function (medplum, event) {
...
 
-    afterAll(() => {
-      getConfig().vmContextBotsEnabled = false;
-    });
+  test('Binary response', async () => {
+    const res = await request(app)
+      .get(`/fhir/R4/Bot/${bots[2].id}/$execute`)
+      .set('Authorization', 'Bearer ' + accessToken);
+    console.log('CODY res.body', res.body);
+    console.log('CODY res.text', res.text);
```
Comment (ThatOneBro) [inline packages/server/src/fhir/operations/execute.test.ts:431]:
```suggestion
```
Cleanup

Comment (mattlong) [inline packages/server/src/fhir/operations/execute.test.ts:431]:
```suggestion
```

**Review by mattlong — APPROVED** [review]
LGTM, just some console statements to remove

**Review by ThatOneBro — APPROVED** [review]

File: packages/server/src/fhir/operations/execute.test.ts (hunk 1 above)
Comment (codyebberson) [reply packages/server/src/fhir/operations/execute.test.ts]: 🤦

Comment (codyebberson) [reply packages/server/src/fhir/operations/execute.test.ts]: 🤦

**Pull request merged**